packages/bpk-component-form-validation/src/BpkFormValidation.js

```diff
@@ -55,6 +63,9 @@ const BpkFormValidation = props => {
     >
       <div className={getClassName('bpk-form-validation__container')}>
         <div className={classNames.join(' ')} {...rest}>
```

Comment:
We've got a nicer syntax now for declaring classNames that means you won't have to do `classNames.join(' ')` (it wasn't available when this component was originally created). If you replace the stuff above it'll make the code a bit nicer:

**Before**

```javascript
const classNames = [getClassName('bpk-form-validation')];
  if (expanded) {
    classNames.push(getClassName('bpk-form-validation--appear'));
  }
  if (isCheckbox) {
    classNames.push(getClassName('bpk-form-validation--is-checkbox'));
  }
  if (className) {
    classNames.push(className);
  }
```

**After**

```javascript
const classNames = getClassName(
  'bpk-form-validation',
  expanded && 'bpk-form-validation--appear',
  isCheckbox && 'bpk-form-validation--is-checkbox',
  className,
);
```

Then later in the render code you can just do `className={classNames}`.

Here's a [working example from another component](https://github.com/Skyscanner/backpack/blob/e393f0a6e9ded9c2f27dd3d7547c4a5e6fc36149/packages/bpk-component-ticket/src/BpkTicket.js#L55).